guides/shipments/user-interface-for-shipments.md

```diff
@@ -0,0 +1,65 @@
+# User interface for shipments
+
+This article explores important elements of the user interface for shipments.
+While Solidus's user interface is entirely customizable, this article is
+modelled after a typical Solidus installation, which would use the standard UI
+elements present in the `solidus_frontend` and `solidus_backend` gems.
+
+## Customer-facing UI
+
+In a typical Solidus store, there is no mention of shipping until checkout.
+
+After the customer enters the shipping address for their order, the UI displays
+any shipping methods that are available for the current order. (Every time that
+the order is changed or updated, the available shipping methods would also be
+updated.)
+
+For more information on how shipping methods are filtered, see [Shipping method
+filters](shipping-method-filters.html.markdown).
+
+The customer must choose a shipping method for each shipment before proceeding
+to the next stage of the checkout process.
+
+Once they have reached the order confirmation page, the shipping cost is shown
+and included in the order total.
+
+<!-- TODO:
+  Add checkout images from a demo Solidus store.
+-->
+
+## Administrator-facing UI
+
+Shipment objects (`Spree::Shipment`) are created during checkout. Multiple
+shipments can be associated with a single order.
+
+Shipments do not have their own dedicated part of the admin UI. Shipments are
+viewable and editable from within an order.
+
+In the admin dashboard. the administrator can update each shipment's shipping
+cost and add tracking codes when editing an order. They can also perform other
+tasks, such as splitting a single shipment into multiple shipments.
+
+In the Solidus demo store, edit any order (from the `/admin/orders` URL) and go
```

Comment:
For completed orders, the shipments tab *is* the default interface. It might feel confusing to users if they think they have to navigate.